### PR TITLE
Feature/bern crt board2

### PR DESCRIPTION
**Pull request by mstancar**
### Description

Put description of request here. Include _Fixes #XXX_ to link to an issue.

_If the full description is in a PR from a different repo, simply link that here, and delete the rest._

### Related Repository Branches

List related branches from other repos here (links to PRs even better!), if not linked above.

### Testing details
_(Note: edit as appropriate.)_
- *Where it was tested*: (e.g. SBN-FD, SBN-ND, DAB, etc.)
- *Run number associated with test*: 
- Other information:

### CODE REVIEW

File: sbn-nd/DAQInterface/configs/standard/ptb01PULL.fcl [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include "ptb01.fcl"
+
+## Pull mode parameters
+    daq.fragment_receiver.request_address:        "227.128.11.129" # -- multicast request address
+    daq.fragment_receiver.multicast_interface_ip: "10.226.36.0" # -- should match the private net
+    daq.fragment_receiver.request_port: 3001   # UDP request port
+    daq.fragment_receiver.request_mode: window # pull mode buffer or window
+<<<<<<< HEAD
```
Comment (aaduszki) [inline sbn-nd/DAQInterface/configs/standard/ptb01PULL.fcl]: Seems something didn't merge right

File: sbn-nd/DAQInterface/run [hunk 2]
```diff
@@ -12,7 +12,22 @@ getStatus "nil"
 # [ $status == 'Existing'] && (kill_daqinterface_on_partition.sh 0; exit 0 )
 
 
-setdaqcomps.sh component01 component02
+#setdaqcomps.sh component01 component02
+<<<<<<< HEAD
```
Comment (aaduszki) [inline sbn-nd/DAQInterface/run]: Not merged properly

**Review by aaduszki — COMMENTED** [review]
Hi, I'm not sure what review is needed from me. The configuration looks messy, but I guess it's some temporary testing version?
I presume addition of binary .root file is accidental?

**Comment by gennadiy-fnal** [comment]
We need to review this PR ASAP.

**Comment by gennadiy-fnal** [comment]
Please resolve merge conflicts in sbn-nd/DAQInterface/configs/standard/ptb01PULL.fcl.

**Review by eyandel — APPROVED** [review]

**Comment by gennadiy-fnal** [comment]
Please update the Description and Test details sections of this PR.

**Review by wbadgett — APPROVED** [review]

File: sbn-nd/DAQInterface/configs/standard/ptb01PULL.fcl (hunk 1 above)
Comment (jsensenig) [reply sbn-nd/DAQInterface/configs/standard/ptb01PULL.fcl]: @aaduszki  It looks like `daq.fragment_receiver.request_window_width` and `daq.fragment_receiver.request_window_offset` are defined twice for some reason. These two parameters aren't yet finalized so any of the 2 redundant parameters can be removed.

Comment (jsensenig) [reply sbn-nd/DAQInterface/configs/standard/ptb01PULL.fcl]: Conflict resolved in this file.

**Comment by jsensenig** [comment]
The merge conflict in `ptb01PULL.fcl` has been resolved.

**Pull request merged**